hawc/apps/common/lookups.py

```diff
@@ -60,9 +60,24 @@ def get_underscore_field_val(self, obj: Any, underscore_path: str) -> Any:
         Returns:
             Any: the desired attribute of the object or child object.
         """
+
+        attr_whitelist = [
```

Comment:
this specific list is endpoint specific; in the base class I'd make maybe rename this `lookup_whitelist`, and then put this specific list in the EndpointByStudyLookup.  Also, change it to a set instead of a list?